src/main/java/teammates/common/util/Const.java

```diff
@@ -445,6 +447,7 @@ private Const() {
 
         // Msq
         public static final int MSQ_MIN_NUM_OF_CHOICES = 2;
+        public static final String EMPTY_MCQ_OPTION = "";
```

Comment:
You shouldn't need this? An empty string is well-understood.

Reply:
Will revert it